config/docker/cros-tast/baseline_cros.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 Collabora Limited
+# Author: Denys Fedoryshchenko <denys.f@collabora.com>
+#
+# This script is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+import paramiko
+import subprocess
+import sys
+
+loglevels = ["crit", "alert", "emerg", "err", "warn"]
+keyfile = '/home/cros-tast/.ssh/id_rsa'
+
+
+def report_lava(testname, result, count):
+    """
+    Report test state/result to LAVA,
```

Comment:
```suggestion
    """ Report test state/result to LAVA
```

Short description should be put right after the triple quotes.

Reply:
Good catch, added also space after short summary, thanks!